roles/inspec/tasks/main.yml

```diff
@@ -29,29 +28,39 @@
     state: directory
   when: inspec.dependency_install_method == 'tar'
 
+- name: populate dependent profile list
+  set_fact:
+    inspec_dependency_profiles: '{{ inspec_dependency_profiles|default([]) + [ inspec.profiles[item].profile_installation ] }}'
```

Comment:
Can you clarify on what you mean by utilizing `inpec.profiles[item].profile_installation|unique`? I  think unique is a filter that operates on lists. `inpec.profiles[item].profile_installation` is meant to be a reference to an entry in inspec.profiles.profile_installation.

What I was trying to do here is set a variable/fact for inspec_dependency_profiles which will contain the list of required references in inspec.profile_installations.
It starts out as an empty list and the list is appended per item in inspec.profiles.
If we take a look at the inspec.profiles variable there are entries
for rhel7,horizon,keystone..... horizon and keystone have the same reference to inspec.profile_installations.openstack_security. This would mean openstack_security
would be duplicate in the list so that is why the unique filter is used.